### PR TITLE
[12.0][ADD] stock_picking_return_restricted_qty: Don't allows you to return more quantity than delivered

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT22466

### CODE REVIEW

File: stock_picking_return_restricted_qty/README.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+#. Create an outgoing or an internal picking.
```
Comment (sergio-teruel) [inline stock_picking_return_restricted_qty/README.rst]: Please take an account incoming pickings.

File: stock_picking_return_restricted_qty/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Return Restricted Qty",
+    "summary": "This module don't allows you to return more products"
+               "than you had bought",
+    "version": "12.0.1.0.0",
+    'development_status': 'Alpha',
+    "license": "AGPL-3",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
```
Comment (sergio-teruel) [inline stock_picking_return_restricted_qty/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

File: stock_picking_return_restricted_qty/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "website": "https://github.com/OCA/stock-logistics-workflow/",
+    "depends": ["stock"],
+    "data": [],
```
Comment (sergio-teruel) [inline stock_picking_return_restricted_qty/__manifest__.py]: Remove empty dictionary key

File: stock_picking_return_restricted_qty/wizard/stock_picking_return.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+class ReturnPicking(models.TransientModel):
+    _inherit = 'stock.return.picking'
+
+    @api.model
+    def default_get(self, fields_list):
+        res = super().default_get(fields_list)
+        if 'product_return_moves' not in res:
+            return res
+        return_line = self.env['stock.return.picking.line']
```
Comment (sergio-teruel) [inline stock_picking_return_restricted_qty/wizard/stock_picking_return.py:19]: Add  a comment explaing what are you doing

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@sergio-teruel Changes done!

File: stock_picking_return_restricted_qty/readme/CONTRIBUTORS.rst [hunk 5]
```diff
@@ -0,0 +1 @@
+* Carlos Roca <carlos.roca@tecnativa.com>
```
Comment (manuelcalerosolis) [inline stock_picking_return_restricted_qty/readme/CONTRIBUTORS.rst]: Write your name under a link to the Tecnativa website like it is here
https://github.com/OCA/web/blob/12.0/web_decimal_numpad_dot/readme/CONTRIBUTORS.rst

**Review by manuelcalerosolis — APPROVED** [review]
Code Review: it's ok
Functional Review : It's Ok (test localy)

**Comment by CarlosRoca13** [comment]
@manuelcalerosolis Changes done!

**Review by rafaelbn — APPROVED** [review]
Functionally tested in runbot :+1: 

@sergio-teruel last review please

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**